modules/specialized-guides/pages/salt/salt-connectivity.adoc

```diff
@@ -0,0 +1,156 @@
+
+# How long before the first keepalive should be sent in seconds. Default 300
+# to send the first keepalive after 5 minutes, OS default (-1) is typically 7200 seconds
+# on Linux see /proc/sys/net/ipv4/tcp_keepalive_time.
+tcp_keepalive_idle: 10
+
+# How many lost probes are needed to consider the connection lost. Default -1
+# to use OS defaults, typically 9 on Linux, see /proc/sys/net/ipv4/tcp_keepalive_probes.
+tcp_keepalive_cnt: 3
+
+# How often, in seconds, to send keepalives after the first one. Default -1 to
+# use OS defaults, typically 75 seconds on Linux, see
+# /proc/sys/net/ipv4/tcp_keepalive_intvl.
+tcp_keepalive_intvl: 10
+----
+
+
+
+
+[[proxies-connectivity]]
+== Proxies Connectivity
+
+[command]``salt-broker`` service is used on {productname} Proxies to forward `salt` traffic between the {productname} Server
+and [command]``salt-minion`` services used on the client side.
+It's possible that [command]``salt-broker`` and all the clients behind it could be affected with the same issue as the clients
+directly connected to the {productname} Server.
+The issue could be fixed with the same parameters as recommended for the minions, but specified in [path]``/etc/salt/broker`` on each {productname} Proxy.
+
+
+The other possible issue which {productname} Proxy can be affected with is the case when the connectivity to the {productname} Server
+is lost for quite long interval, so the clients behind it started to retry the authentication to the [command]``salt-master`` service on the {productname} Server.
+This situation could be potentially dangerous as it could lead to collecting large amount of ZeroMQ messages with authentication attemps
+in the interal buffer of ZeroMQ sockets used inside the [command]``salt-broker`` service, so that on restoring the connection to the [command]``salt-master``
+all of the messages will be pushed to it. It could lead to the issues on {productname} Server side with [command]``salt-master`` service,
+as it could be impossible to serve all the cached requests in apropriate time or even to the complete denial of the service.
+
+To avoid such situation a set of extra parameters was introduced, the most important one is ``wait_for_backend``, which sould be set to ``True``,
+it prevents opening the sockets for the clients behind the proxy while the connectivity to the [command]``salt-master`` service is not established.
+In this case the messages from the clients are not collected in the internal buffers.
+``drop_after_retries`` is setting the number of retries before closing the sockets to drop the cached messages.
+The other parameters could help to fine tune the behaviour for the environment.
+
+[NOTE]
+====
+Setting timouts, intevals and ``drop_after_retries`` to lower values are making [command]``salt-broker`` service more agressive on detecting
+the connection loss to the [command]``salt-master``, so that it puts the clients behind the proxy to the conditions closer as they are connected
```

Comment:
```suggestion
Setting timeouts, intervals and ``drop_after_retries`` to lower values are making [command]``salt-broker`` service more aggressive on detecting the connection loss to the [command]``salt-master``, so that it puts the clients behind the proxy to the conditions closer as they are connected
```